src/Okta.Sdk.IntegrationTests/ApplicationScenarios.cs

```diff
@@ -0,0 +1,553 @@
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).IdpIssuer.Should().Be("http://www.okta.com/${org.externalKey}");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).Audience.Should().Be("asdqwe123");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).Recipient.Should().Be("http://testorgone.okta");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).Destination.Should().Be("http://testorgone.okta");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).SubjectNameIdTemplate.Should().Be("${user.userName}");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).SubjectNameIdFormat.Should().Be("urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).ResponseSigned.Should().BeTrue();
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AssertionSigned.Should().BeTrue();
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).SignatureAlgorithm.Should().Be("RSA_SHA256");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).DigestAlgorithm.Should().Be("SHA256");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).HonorForceAuthn.Should().BeTrue();
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AuthnContextClassRef.Should().Be("urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).SpIssuer.Should().BeNull();
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).RequestCompressed.Should().BeFalse();
+
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AttributeStatements.Should().HaveCount(1);
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AttributeStatements.First().Type.Should().Be("EXPRESSION");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AttributeStatements.First().Name.Should().Be("Attribute");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AttributeStatements.First().Namespace.Should().Be("urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified");
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AttributeStatements.First().Values.Should().HaveCount(1);
+                ((SamlApplicationSettingsSignOn)retrievedSettings.SignOn).AttributeStatements.First().Values.First().Should().Be("Value");
+            }
+            finally
+            {
+                await client.Applications.DeactivateApplicationAsync(createdApp.Id);
+                await client.Applications.DeleteApplicationAsync(createdApp.Id);
+            }
+        }
+
+        /// <summary>
+        /// FAIL: GroupName expected to be null or empty but is not
```

Comment:
It sounds like these are bugs with our example code on developer.okta.com. In that case, file it as an issue on https://github.com/okta/okta.github.io and change the values here to get the test working. 👍

Reply:
https://github.com/okta/okta.github.io/issues/2141